api.py

```diff
@@ -235,16 +229,18 @@ def get(self, user_id):
                 SELECT user_id FROM users_convos WHERE convo_id=%s
                 '''
 
+        # for a convo id, which the same user is a member of, select all the user_id associated with it
+
         convos = {}
         for convo_id in query:
```

Comment:
Good catch on that bug, but I think a better fix is getting rid of the indexing within each time we access convo_id tuple and instead making it
`for convo_id, in query:` 
That will access the first element in each tuple going through the query list. I think it makes the code more ETU.